File: C#/Multiply.cs
public class Solution {
    public string Multiply(string num1, string num2) {
        
        List<ulong> Numb2 = new List<ulong>();
        
        for (int i = 0; i < num2.Length; i++)
        {
            Numb2.Add(Convert.ToUInt64(new String(num2[i], 1)));
        }
        
        ulong Numb1 = Convert.ToUInt64(num1);
        
        Console.WriteLine("Number 1: " + Numb1);
        Console.Write("\n");
        Console.Write("Number 2: ");
        for (int i = 0; i < Numb2.Count; i++)
        {
            Console.Write(Numb2[i] + ", ");
        }
        Console.Write("\n");
        
        double Output = 0;
        
        for (int i = Numb2.Count - 1; i > -1; i--)
        {
            Output += Numb1 * Numb2[i] * Math.Pow(10, Numb2.Count - 1 - i);
            
            // Debug
            Console.WriteLine(Numb1 * Numb2[i] * Math.Pow(10, Numb2.Count - 1 - i));
        }

        return Output.ToString();
    }
}

File: Python/multiply.py
class Solution:
    def multiply(self, num1: str, num2: str) -> str:

        # Default
        if (num1 == "0") | (num2 == "0"):
            return "0"

        numDict = {
        "0": 0,
        "1": 1,
        "2": 2,
        "3": 3,
        "4": 4,
        "5": 5,
        "6": 6,
        "7": 7,
        "8": 8,
        "9": 9,
        }

        def convtStrToInt(strNum):
            num = 0
            for char in strNum:
                num *= 10
                num += numDict[char]
            return num
            
        numb1 = convtStrToInt(num1)
        numb2 = convtStrToInt(num2)
        
        prod = numb1 * numb2

        dictKeys = list(numDict.keys())

        def convtIntToStr(num):
            str = ""
            while (num > 0):
                str = dictKeys[num % 10] + str
                num = num // 10
            return str

        return convtIntToStr(prod)